text.cabal

```diff
@@ -170,6 +170,7 @@ library
     Data.Text.Internal.Read
     Data.Text.Internal.Search
     Data.Text.Internal.StrictBuilder
+    Data.Text.Internal.Transformation
```

Comment:
Let's stop pollution of the global namespace. Please make this internal module only.

Reply:
Done in c3fe10febc949b74aed364c68aa272d3821fd70e